test/protocol/integration/exchange/curveStEthExchangeAdapterForked.spec.ts

```diff
@@ -0,0 +1,140 @@
+import "module-alias/register";
+
+import { BigNumber, ContractTransaction } from "ethers";
+
+import { Address } from "@utils/types";
+import { Account } from "@utils/test/types";
+import { MAX_UINT_256 } from "@utils/constants";
+import DeployHelper from "@utils/deploys";
+import { ether } from "@utils/index";
+import {
+  addSnapshotBeforeRestoreAfterEach,
+  getAccounts,
+  getForkedTokens,
+  getSystemFixture,
+  getWaffleExpect,
+  initializeForkedTokens
+} from "@utils/test/index";
+
+import { SystemFixture } from "@utils/fixtures";
+import { CurveStEthExchangeAdapter, CurveEthStEthExchangeMock } from "@utils/contracts";
+
+import { StandardTokenMock } from "@typechain/StandardTokenMock";
+import dependencies from "@utils/deploys/dependencies";
+
+const expect = getWaffleExpect();
+
+describe("CurveStEthExchangeAdapter [ @forked-mainnet ]", () => {
```

Comment:
Added a forked mainnet test - is there a convention for naming files for forked mainnet tests?